README.md

```diff
@@ -3,6 +3,10 @@
 [![Go Report Card](https://goreportcard.com/badge/tektoncd/operator)](https://goreportcard.com/report/tektoncd/operator)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/6548/badge)](https://bestpractices.coreinfrastructure.org/projects/6548)
 
+<p align="center">
+<img width="250" height="300"  src="/image/icon.png" alt="Operator Icon" title="Operator Icon"></img>
+</p>
```

Comment:
Why is this in `html` and not using `markdown` ? 🤔 I think it currently doesn't display it correctly (because of the `/`). If we want to keep this, it should be `image/icon.png`.

Reply:
Note that it seems all other project have the image on root named like `tekton-{component}.png`.

Reply:
https://github.com/sm43/tektoncd-operator/tree/add-icon-to-readme#readme